rest-api-spec/src/main/resources/rest-api-spec/api/security.saml_sp_metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+  "security.saml_sp_metadata":{
```

Comment:
Thoughts on expanding sp to service_provider for better discoverability/readability?